anaconda.py

```diff
@@ -291,16 +291,16 @@ def start_debugger(signum, frame):
         flags.dirInstall = True
 
     # Set up logging as early as possible.
-    import logging
-    from pyanaconda import anaconda_log
-    anaconda_log.init()
-    anaconda_log.logger.setupVirtio()
+    from pyanaconda import anaconda_logging
```

Comment:
I was curious what we log in `flags.py` and maybe the function `can_touch_runtime_system` could be moved somewhere else?